internal/completions/httpapi/codecompletion.go

```diff
@@ -48,8 +48,12 @@ func allowedCustomModel(model string) string {
 	case "fireworks/starcoder",
 		"fireworks/starcoder-16b",
 		"fireworks/starcoder-7b",
+		"fireworks/starcoder2-15b",
+		"fireworks/starcoder2-7b",
 		"fireworks/" + fireworks.Starcoder16b,
 		"fireworks/" + fireworks.Starcoder7b,
+		"fireworks/" + fireworks.StarcoderTwo15b,
+		"fireworks/" + fireworks.StarcoderTwo7b,
```

Comment:
```suggestion
		"fireworks/starcoder2-15b",
		"fireworks/starcoder2-7b",
		"fireworks/" + fireworks.Starcoder16b,
		"fireworks/" + fireworks.Starcoder7b,
```

If you add the virtual model names, let's make sure that clients use those to connect exclusively.